.github/workflows/tests.yml

```diff
@@ -0,0 +1,45 @@
+name: tests
+
+on: push
```

Comment:
I think we will need something in the vein of on push and on PR. We want it triggered for new PRs and any time a PR is updated.

Reply:
I think opened, reopened, edited, sychronized?

Reply:
Good point! I'll add the `pull_request`.

About the PR events: it looks like opened, reopened, and synchronized are [on by defaut](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request). Correct me if I'm wrong, but edited seems to also include modification like "edit the PR title": do we also want that to retrigger the tests?

Reply:
I don't actually want it to run for editing the title, but I want it to run for changing the base branch and I don't think there's a separate action for that?

Reply:
I couldn't find another type either.
I've pushed the changes :)